ts-defs/index.d.ts

```diff
@@ -5,7 +5,7 @@ interface ClientFunctionOptions {
      *  Contains functions, variables or objects used by the client function internally.
      *  Properties of the `dependencies` object will be added to the client function's scope as variables.
      */
-    dependencies?: {[key: string]: Function},
+    dependencies?: {[key: string]: any},
```

Comment:
We should do the same for functions with `filterFn` (like `filter`, `find` etc..)